docs/NNNN-schema-registry.md

```diff
@@ -0,0 +1,195 @@
+Extension Name: NNNN-schema-registry
+Authors: P. Cornwell, D. Granville
+Minimum OCFL Version: 1.0
+OCFL Community Extensions Version: 1.0
+Extension Version: 0.1
+Obsoletes: n/a
+Obsoleted by: n/a
+
+## Overview
+An OCFL object will typically contain metadata serialised in JSON or XML files. These must currently conform to one or more schemata referenced as external JSONSchema/XSD/DTD files.
+
+In order for an OCFL root to represent a self-consistent repository, the specific versions of each schema referenced must be available in order to validate the content of OCFL objects. It is currently assumed that such schemata are maintained independently in external files and remain accessible via URLs. However, maintaining a local copy of schemata within the OCFL root is essential where long-term preservation is a goal. External online references must not be relied upon in the long-term, due to the possibility of evolution of technical ecosystems and of organizational change.
+
+This extension stores a single copy of each schema referenced by the content of OCFL objects within the root: it avoids the need to store a copy of schemata redundantly in each OCFL object. It also provides a convenient reference point for archive management software and future users inspecting the contents of the OCFL object.
+
+We describe a standardised layout for a schema directory (hereinafter schema registry), where this extension stores all schemata referenced throughout the OCFL root. This schema registry must be implemented by creating and maintaining the following items:
+
+* A config.json file containing configuration parameters for the registry.
+* The 'schemata' directory containing a reference copy of each schema in use.
+* The 'schema_inventory.json' file providing an index of and checksums for the stored schemata. 
+* A sidecar file 'schema_inventory.json.sha512' (or other configured digest) containing the checksum of schema_inventory.json, in the same manner as OCFL inventory files. 
+
+Example file content and structures are shown in the Examples section below.
+
+## Parameters
+
+Configuration is done by setting values in config.json at the top level of the extension's directory. The keys expected are:
+
+* Name: extensionName
+  * Description: String identifying the extension.
+  * Type: String
+  * Constraints: Must be "NNNN-schema-registry".
+  * Default: "NNNN-schema-registry"
+
+* Name: extensionVersion
+  * Description: String identifying extension version.
+  * Type: String
+  * Constraints: Must be a valid published version of this specification.
+  * Default: "0.1"
```

Comment:
Remove `extensionVersion` block per discussion above that extensions aren't versioned, they are replaced with new extensions

Reply:
will do as mentioned elsewhere